### DISCUSSION TITLE
Failing Issues When Running the `yarn test` command at my end

### DISCUSSION THREAD

**Discussion by justice-asare** [perspective-dev/perspective#2194] [General]
Hello!

Error encountered after running yarn test command for Python development:

yarn run v1.22.19
$ node scripts/test.js
Python 3.10.6
ERROR: usage: __main__.py [options] [file_or_dir] [file_or_dir] [...]
__main__.py: error: unrecognized arguments: --cov-report=xml --cov-branch --cov=perspective
  inifile: /home/justice/finos/perspective/python/perspective/pyproject.toml
  rootdir: /home/justice/finos/perspective/python/perspective

Command failed: cd /home/justice/finos/perspective/python/perspective && python3 -m pytest --noconftest perspective/tests/client_mode --junitxml=python_junit.xml --cov-report=xml --cov-branch --cov=perspective --disable-pytest-warnings
error Command failed with exit code 1.

**Comment by timkpaine** [comment]
You haven't installed the development dependencies, in this case you're missing `pytest-cov`. There is a helper function to install all development dependencies, `yarn _requires_python`
